Trimble.Modus.Components/Controls/Toast/TMToastContents.xaml.cs

```diff
@@ -0,0 +1,160 @@
+using Microsoft.Maui.Platform;
+using Trimble.Modus.Components.Enums;
+using Trimble.Modus.Components.Popup.Services;
+
+namespace Trimble.Modus.Components.Controls.Toast;
+
+public partial class TMToastContents : Popup.Pages.PopupPage
+
+{
+    public ImageSource LeftIconSource { get; set; }
+    public string Message { get; set; }
+
+    public string RightIconText { get; set; }
+
+    public ImageSource RightIconSource { get; set; }
+
+    public double ToastWidthRequest { get; set; }
+
+    public Color ToastTheme { get; set; }
+
+    public Color TextColor { get; set; }
```

Comment:
Are all these properties need to be public? I don't see any other classes uses these. Can we have them as private? Can we still access those in the .xaml file?

Reply:
Yes changing it private or internal makes it not accessible to xaml